plugins/forward/alerta_forward.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from alerta.plugins import PluginBase
+from alertaclient.api import Client
+
+LOG = logging.getLogger('alerta.plugins.forward')
+
+FORWARD_URL = app.config.get('FORWARD_URL')
+FORWARD_API = app.config.get('FORWARD_API')
+
+class ForwardAlert(PluginBase):
+
+    def pre_receive(self, alert):
+        return
```

Comment:
The `pre-receive()` hook needs to return the alert like this ...
https://github.com/alerta/alerta/blob/master/alerta/plugins/__init__.py#L20